docs/content/guides/concurrency-scheduling.md

```diff
@@ -0,0 +1,240 @@
+   a service can take. We'll set it to `20` as an example.
+5. `priority_label_key`: This field specifies the label that is used to
+   determine the priority. We will leave the label as it is.
+6. `tokens_label_key`: This field specifies the label that is used to determine
+   tokens. We will leave the label as it is.
+7. `workload_label_key`: This field specifies the label that is used to
+   determine the workload. We will leave the label as it is.
+8. `control_point`: It can be a particular feature or execution block within a
+   service. We'll use `concurrency-scheduling-feature` as an example.
+
+Here is how the complete values file would look:
+
+```yaml
+{@include: ./assets/concurrency-scheduling/values.yaml}
+```
+
+The last step is to apply the policy using the following command:
+
+<CodeBlock language="bash"> aperturectl cloud blueprints apply
+--values-file=concurrency-scheduling-test.yaml </CodeBlock>
+
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
+
+Next, we'll proceed to run an example to observe the newly implemented policy in
+action.
+
+## Concurrency Scheduling in Action
+
+Begin by cloning the
+[Aperture JS SDK](https://github.com/fluxninja/aperture-js). Look for the
+`concurrency_scheduler_example.ts` in the example directory within the SDK.
+
+Switch to the example directory and follow these steps to run the example:
+
+1. Install the necessary packages:
+   - Run `npm install` to install the base dependencies.
+   - Run `npm install @fluxninja/aperture-js` to install the Aperture SDK.
+2. Run `npx tsc` to compile the TypeScript example.
+3. Run `node dist/concurrency_scheduler_example.js` to start the compiled
+   example.
+
+Once the example is running, it will prompt you for your Organization address
+and API Key. In the Aperture Cloud UI, select the Aperture tab from the sidebar
+menu. Copy and enter both your Organization address and API Key to establish a
+connection between the SDK and Aperture Cloud.
+
+## Monitoring concurrency Scheduling Policy
+
+After running the example for a few minutes, you can review the telemetry data
+in the Aperture Cloud UI. Navigate to the Aperture Cloud UI, and click the
+`Policies` tab located in the sidebar menu. Then, select the
+`concurrency-scheduling-test` policy that you previously created.
+
+Once you've clicked on the policy, you will see the following dashboard:
+
+![Workload](./assets/managing-quotas/workloads.png)
+
+The two panels above provide insights into how the policy is performing by
+monitoring the number of accepted and rejected requests along with the
+acceptance percentage.
+
+![Request](./assets/managing-quotas/request-metrics.png)
+
+The panels above offer insights into the request details, including their
+latency.
+
+![Queue](./assets/managing-quotas/queue.png)
+
+These panels display insights into queue duration for `workload` requests and
+highlight the average of prioritized requests that moved ahead in the queue.
```

Comment:
Preemption for each token measured as the average number of tokens a request belonging to that workload gets preempted in the queue